### DISCUSSION TITLE
VPS on Ionos and Kamatera stopped working 26/12/24. Lightsail working fine

### DISCUSSION THREAD

**Discussion by sieade245** [Ysurac/openmptcprouter#3722] [Q&A] [answered]
Hi everyone. Just looking to see if any others are experiencing this issue before I spent any more time fault finding (so far spent over 10 hours).

Had a working OMR for the past month using an Ionos basic VPS 1 core/ 1GB RAM. Bonding 3 connections, 2x 5G + fibre. Not really had any issues with this. Haven't made any major changes to the config for a few weeks after it was all setup. I also set up alternative VPS on Amazon Lightsail and Kamatera (with similar hardware specs).

2 days ago, my VPS on Ionos stopped working properly. On the router status page, on the VPS icon,  I get various DNS related errors such as "xxx wasn't able to get public IP in 1 second" and "no output". I tried the usual first line steps, rebooting the VPS and the router and all of my modems with no effect. I then spent the 10 hours mentioned earlier changing literally every setting in the router to see if I could fix it but no luck. Looking back I really should have just left this alone as it was working fine before.

So then I tried my alternative VPSs. Same issue on Kamatera. Lightsail works fine though. I made no changes to my router, I just change the VPS IP address and the access key. The Lightsail VPS changes over almost instantly and is fully working.

Thinking maybe it was something I had done on my router, which had also affected the VPS somehow, I purchased a second Ionos server today and set up the VPS from scratch using the 6.6 install script. I tried to use this with the router but received the exact same errors as before. This made me feel a bit better as its unlikely to be a VPS issue.

I thought it would be a good idea to set up the router from scratch again just in case. I'm on X86-64 on a proxmox system. I rolled back to a snapshot from a month ago when I first set it up. Right off the bat, both Ionos servers and the Kamatera server still don't work with the same public IP address error. These were working when I took this snapshot (obviously not the 2nd Ionos one I just bought). The lightsail server again working fine no issues at all.

Double checked the VPS provider firewall settings for 65000-65535 TCP forwarding although I hadn't made any changes to any of this on any VPS. 

That kind of leaves me with the conclusion that something has happened to the VPS backend on Ionos and Kamatera. Seems to be the same problem on both providers. I realise its Christmas and everyone's probably on holiday there so was going to leave it a few days before I contacted them to see if they have made any changes, or potentially if they have blocked this kind of usage. If this is the case, I haven't received any information regarding this AFAIK. 

Does anyone else in the community have VPS deployed on Ionos/ Kamatera and are you having similar issues?

**Comment by Ysurac** [comment]
Even with the public IP address error, it was working or not ?
I had a SSL certificate problem on IP detection, should not be a real issue as there is workaround but...
I will test with my IONOS VPS asap.

**Reply by sieade245** [reply]
No I had no external access from the ionos vps at all. Couldn't even ping Google IP address directly without using dns. Something I did notice. If I change any settings in the wizard and save changes. About 6 pings do get through (possibly using the default route?) but then it stops working again. Again this was all working fine before the 26th and the lightsail VPS works fine with zero changes made to the config

**Comment by sieade245** [comment]
An update on this today 01/01/25. I swapped back to Ionos VPS. Made no changes apart from ticked Ionos as master VPS in settings wizard. It is working again on Ionos, the public IP address error has gone and MPTCP routing appears ok. Kamatera is still not working though with the same error message. As I've changed nothing my end, I still think it is a provider issue.

**Comment by sieade245** [comment, marked as answer]
Update today 02/01/25. Kamatera also now working again. Not sure what caused this.
